.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "LISA Development",
+	"dockerFile": "../Dockerfile.linux",
+	"context": "..",
```

Comment:
does it only support linux container?

Reply:
No, it should work with any valid Docker file.
The linux container works with both Windows and Linux, hence used linux.
We could add both, it will look like the below 

<img width="612" height="82" alt="image" src="https://github.com/user-attachments/assets/95330c33-972d-4785-b3ee-86bb307eea51" />
<img width="296" height="117" alt="image" src="https://github.com/user-attachments/assets/2af8a6c3-dedd-4d5a-b5a2-f49cd3b8e3e2" />